src/style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block-snake {
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex: 1 1 33.33%;
+}
+
+.block-snake:first-child {
+  background-color: #f00;
+}
+
+.block-snake:nth-child(2) {
+  background-color: #c00;
+}
+
+.block-snake:nth-child(3) {
+  background-color: #900;
+}
+
+.block-snake:nth-child(4) {
+  background-color: #600;
+  order: 2;
+}
+
+.block-snake:nth-child(5) {
+  background-color: #300;
+  order: 1;
+}
+
+.block-snake:nth-child(6) {
+  background-color: #000;
+}
+
+@media (max-width: 750px) {
+  .block-snake {
+    flex: 50%;
+  }
+
+  .block-snake:nth-child(3),
+  .block-snake:nth-child(5) {
+    order: 3;
+  }
+
+  .block-snake:nth-child(6) {
+    order: 4;
+  }
+}
+
+@media (max-width: 450px) {
+  .block-snake {
+    min-width: 300px;
+  }
+
+  .block-snake:nth-child(3) {
+    order: 1;
+  }
+}
```

Comment:
Wrong breakpoints. Look at the screenshots in the task and fix this mistake, please